packages/client-config/src/client-config-contributor/client_config_contributor_v1.ts

```diff
@@ -112,7 +112,13 @@ export class AuthClientConfigContributor implements ClientConfigContributor {
       authOutput.payload.passwordPolicyMinLength ||
       authOutput.payload.passwordPolicyRequirements
     ) {
-      authClientConfig.auth.password_policy = {};
+      authClientConfig.auth.password_policy = {
+        min_length: 8, // This is the default.
+        require_lowercase: false,
+        require_numbers: false,
+        require_symbols: false,
+        require_uppercase: false,
```

Comment:
This doesn't seem to match the defaults mentioned in our [documentation](https://docs.amplify.aws/react/build-a-backend/auth/manage-users/manage-passwords/#override-default-password-policy).

Reply:
I added comment in https://github.com/aws-amplify/amplify-backend/pull/1751/commits/aa4e65a6a7d4cb24cf2de908ed64a8045a10ee51 .
For these we're setting them to false instead of undefined and this is still re-computed below to honor what construct emits (using defaults or not).

Reply:
So effectively this https://github.com/aws-amplify/amplify-backend/blob/10955d9c373a0836cd986acfd2836c77da56b933/packages/auth-construct/src/defaults.ts#L34-L40. will flow here and produce expected outcome if it hasn't been overridden in backend definition.